libs/dh/shared/util-application-insights/src/lib/dh-application-insights.service.ts

```diff
@@ -55,11 +42,37 @@ export class DhApplicationInsights {
   /**
    * Initialize Application Insights
    */
-  init(): void {
-    if (this.appInsights.appInsights.isInitialized()) {
+  async init() {
+    if (this.appInsights?.appInsights.isInitialized()) {
       return;
     }
 
+    const { ApplicationInsights } = await import(
+      /* webpackChunkName: "application-insights" */
+      '@microsoft/applicationinsights-web'
+    );
+
+    const { AngularPlugin } = await import(
+      /* webpackChunkName: "application-insights-angular-plugin" */
```

Comment:
nitpick: doesn't look like a new chunk with this name gets generated. It's probably bundled in the `application-insights` chunk 🤔

Reply:
I saw this as well, lets leave it :-)